### PR TITLE
Uikit5

### PR DESCRIPTION
**Pull request by gorgeousvlad**

### CODE REVIEW

**Review by NikitaCG — APPROVED** [review]

**Review by yuberdysheva — APPROVED** [review]

File: src/blocks/PromoFeaturesBlock/PromoFeaturesBlock.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@ $block: '.#{$ns}PromoFeaturesBlock';
         background-color: inherit;
 
         &_grey {
-            background-color: var(--yc-color-base-misc);
+            background-color: var(--g-color-base-misc-light);
```
Comment (Kyzyl-ool) [inline src/blocks/PromoFeaturesBlock/PromoFeaturesBlock.scss:31]: We didn't have `--g-color-base-misc` variable? I wonder how this variable may change for dark theme

Comment (gorgeousvlad) [reply src/blocks/PromoFeaturesBlock/PromoFeaturesBlock.scss:31]: "light" here means "pale" or "not heavy", dark theme supports this color, you can switch to to see it.

File: src/editor/components/AddBlock/AddBlock.scss [hunk 2]
```diff
@@ -23,8 +23,8 @@ $block: '.#{$ns}add-block';
         width: $buttonWidth;
         height: $buttonHeight;
 
-        color: var(--g-color-text-inverted-primary);
-        background-color: var(--pc-color-base-neon);
+        color: var(--g-color-text-primary);
+        background-color: var(--pc-editor-base-color);
```
Comment (yuberdysheva) [inline src/editor/components/AddBlock/AddBlock.scss:27]: why don't we use colors from uikit for editing?

File: src/editor/styles/root.scss [hunk 3]
```diff
@@ -2,6 +2,6 @@ body {
     --pc-editor-header-height: 48px;
     --pc-editor-divider-width: 12px;
     --pc-editor-base-color: var(--g-color-base-brand);
-    --pc-editor-control-color: var(--g-color-private-yellow-500-solid);
+    --pc-editor-control-color: var(--g-color-private-yellow-400-solid);
```
[the remark this answers is not in the document]
Comment (gorgeousvlad) [reply src/editor/styles/root.scss:5]: Yuri gave advice to use it on today meeting

File: src/editor/components/AddBlock/AddBlock.scss (hunk 2 above)
Comment (gorgeousvlad) [reply src/editor/components/AddBlock/AddBlock.scss:27]: `--pc-editor-base-color` contains color form uikit, I just created this variable for comfortable working with editor colors

**Review by yuberdysheva — APPROVED** [review]

**Pull request merged**